lock-api/src/main/java/com/palantir/lock/client/TransactionStarter.java

```diff
@@ -79,6 +80,25 @@ StartIdentifiedAtlasDbTransactionResponse startIdentifiedAtlasDbTransaction() {
         }
     }
 
+    List<StartIdentifiedAtlasDbTransactionResponse> startIdentifiedAtlasDbTransactionBatch(int count) {
```

Comment:
There could be a world here where we return Optionals though: it feels almost as if the layer above is better placed to unlock these in an efficient manner (without requiring a request to timelock). But it sure is a rather leaky API